com.unity.render-pipelines.core/Runtime/Common/CollectionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// A set of extension methods for collections
+    /// </summary>
+    public static class CollectionExtensions
+    {
+        /// <summary>
+        /// Removes a given count from the back of the given list
+        /// </summary>
+        /// <param name="inputList">The list to be removed</param>
+        /// <param name="elementsToRemoveFromBack">The number of elements to be removed from the back</param>
+        public static void RemoveBack<T>([NotNull] this IList<T> inputList, int elementsToRemoveFromBack)
```

Comment:
Can you instead implement a `RemoveRange` to be consistent with existing `List.RemoveRange`?

If you need absolutely a `RemoveBack`, you can add another method as syntaxic sugar.

Reply:
Changed to `RemoveRange`